pkg/model/runtime.go

```diff
@@ -56,6 +56,15 @@ func (m *BackstageModel) setRuntimeObject(object RuntimeObject) {
 	m.RuntimeObjects = append(m.RuntimeObjects, object)
 }
 
+func (m *BackstageModel) getRuntimeObjectByType(object RuntimeObject) RuntimeObject {
```

Comment:
I was wondering what would happen if we had multiple objects of the same type, but noticed that this method was only used in the tests. How about moving it there to avoid confusion?

Reply:
Well, current logic does not consider having multiple RuntimeObjects...
Not sure I understand where and why to move this method?

Reply:
Ok, I think I read too fast and have confused `RuntimeObject` with `runtime.Object` :)